### PR TITLE
Dev/vivek

### PR DESCRIPTION
**Pull request by Vivekanand15**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,7 +1,10 @@
...
-}
-export default App;
\ No newline at end of file
+export default ()=>{
+    return (
+        <div>
+            <Search/>
+        </div>
+    );
+};
```
Comment (ShivaniPR) [inline src/App.js]: Add EOF

File: src/component/Search.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                </div>
+            </div>
+            <div className='ui celled list'>
+                {resultRender}
+            </div>
+        </div>
+    );
+}
+
+export default Search;
```
Comment (ShivaniPR) [inline src/component/Search.js:64]: Add EOF

File: src/component/Search.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+                    action:'query',
+                    list:'search',
+                    origin:'*',
+                    format:'json',
+                    srsearch:term,
+                }
+            });
+            setResults(data.query.search);
+        };
+        const timeoutid=setTimeout(() => {
```
Comment (ShivaniPR) [inline src/component/Search.js:35]: What is the need to use setTimeout?

File: src/component/Search.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+        return (
+            <div key={result.pageid} className="item">
+                 <div className="right floated content">
+                    <a className="ui button" href={`https://en.wikipedia.org?curid=${result.pageid}`}>Go</a>
+                </div>
+                <div className="content">
+                    <div className="headers">
+                        {result.title}
+                    </div>
+                    <span dangerouslySetInnerHTML={{__html:result.snippet}} ></span>
```
Comment (ShivaniPR) [inline src/component/Search.js:17]: What is dangerouslySetInnerHTML?

**Review by ShivaniPR — CHANGES_REQUESTED** [review]

File: src/component/Search.js (hunk 4 above)
Comment (Vivekanand15) [reply src/component/Search.js:17]: dangerouslySetInnerHTML is used to set inner HTML for the page

File: src/component/Search.js (hunk 3 above)
Comment (Vivekanand15) [reply src/component/Search.js:35]: the purpose of using setTimeout  is to wait for time. In this we want to wait some amount of time if user stop pressing the key then make search

**Pull request merged**